hoodie-spark/src/main/scala/com/uber/hoodie/AvroConversionUtils.scala

```diff
@@ -53,6 +75,183 @@ object AvroConversionUtils {
     }
   }
 
+  /**
+    * Returns a converter function to convert row in avro format to GenericRow of catalyst.
```

Comment:
copied from spark-avro as these were non-public APIs

Reply:
Nice :) I tried to write this util last year and couldn't do it, great to have it imported from spark

Reply:
Added source details/line. 
Adding test-cases in TestKafkaAvroSource  and TestDFSSource to cover this path.